ecommerce/coupons/tests/mixins.py

```diff
@@ -197,8 +194,19 @@ def mock_course_runs_endpoint(
             discovery_api_url,
             course_run.id if course_run else 'course-v1:test+test+test'
         )
-        httpretty.register_uri(
-            httpretty.GET, course_run_url_with_key,
+        responses.add(
+            responses.GET, course_run_url_with_key,
+            body=json.dumps(course_run_info['results'][0]),
```

Comment:
It seems we could use `json` param instead of `body` to omit the encoding step, or I've missed something?